ci/prow/config.yaml

```diff
@@ -105,6 +109,8 @@ presets:
   env:
   - name: GOOGLE_APPLICATION_CREDENTIALS
     value: /etc/release-account/service-account.json
+  - name: E2E_CLUSTER_REGION
```

Comment:
do you also want to change the `# backups service account`?

Reply:
No, it doesn't run E2E tests.

/hold cancel